### ISSUE TITLE
Button combine doest work.

### ISSUE THREAD

**Issue by Delavore** [kcharvi/Deep-Learning-Project-Labs#1]
Hello. I love your app but when i hit "combine" button nothing happen. Can you help me find problem

**Comment by REDDITARUN** [comment]
Hi, Could you be more specific about which "Combine" Button u talking about?

**Comment by Delavore** [comment]
Yes. About this button![Screenshot_20220823-180755.jpg](https://user-images.githubusercontent.com/92951959/186194953-5b37f528-cbe8-41ed-8e4e-66671773a602.jpg)
